File: Payhub - SDK/java_SDK/PayHubSDK/src/main/java/com/payhub/ws/model/BillingReferences.java
package com.payhub.ws.model;
import com.fasterxml.jackson.annotation.JsonInclude;

@JsonInclude(JsonInclude.Include.NON_NULL)  
public class BillingReferences {
	
	private static final long serialVersionUID = -9171573799378897702L;
    private String cardObscured;
    private String tokenizedCard;
    private long customerId;
    private String customerCardId;    
    private long customerBillId;
	public long getSerialVersionUID() {
		return serialVersionUID;
	}

	public String getCardObscured() {
		return cardObscured;
	}
	public void setCardObscured(String cardObscured) {
		this.cardObscured = cardObscured;
	}
	public String getTokenizedCard() {
		return tokenizedCard;
	}
	public void setTokenizedCard(String tokenizedCard) {
		this.tokenizedCard = tokenizedCard;
	}
	public long getCustomerId() {
		return customerId;
	}
	public void setCustomerId(long customerId) {
		this.customerId = customerId;
	}
	public String getCustomerCardId() {
		return customerCardId;
	}
	public void setCustomerCardId(String customerCardId) {
		this.customerCardId = customerCardId;
	}
	public long getCustomerBillId() {
		return customerBillId;
	}
	public void setCustomerBillId(long customerBillId) {
		this.customerBillId = customerBillId;
	}
    
    

}

File: Payhub - SDK/php_SDK/com/payhub/ws/model/BillingReferences.php
<?php

/**
 * Created by PhpStorm.
 * User: agustin
 * Date: 23/07/2015
 * Time: 12:32
 */
class BillingReferences
{
    public $cardObscured;
    public $tokenizedCard;
    public $customerId;
    public $customerCardId;
    public $customerBillId;

    /**
     * @return mixed
     */
    public function getCardObscured()
    {
        return $this->cardObscured;
    }

    /**
     * @param mixed $cardObscured
     */
    public function setCardObscured($cardObscured)
    {
        $this->cardObscured = $cardObscured;
    }

    /**
     * @return mixed
     */
    public function getTokenizedCard()
    {
        return $this->tokenizedCard;
    }

    /**
     * @param mixed $tokenizedCard
     */
    public function setTokenizedCard($tokenizedCard)
    {
        $this->tokenizedCard = $tokenizedCard;
    }

    /**
     * @return mixed
     */
    public function getCustomerId()
    {
        return $this->customerId;
    }

    /**
     * @param mixed $customerId
     */
    public function setCustomerId($customerId)
    {
        $this->customerId = $customerId;
    }

    /**
     * @return mixed
     */
    public function getCustomerCardId()
    {
        return $this->customerCardId;
    }

    /**
     * @param mixed $customerCardId
     */
    public function setCustomerCardId($customerCardId)
    {
        $this->customerCardId = $customerCardId;
    }

    /**
     * @return mixed
     */
    public function getCustomerBillId()
    {
        return $this->customerBillId;
    }

    /**
     * @param mixed $customerBillId
     */
    public function setCustomerBillId($customerBillId)
    {
        $this->customerBillId = $customerBillId;
    }

    public static function fromArray($data){
        if(!is_null($data)) {
            $billReference = new BillingReferences();
            foreach ($data as $key => $value) {
                if (property_exists(get_class($billReference), $key)) {
                    $billReference->{$key} = $value;
                }
            }
            return $billReference;
        }
    }

}